Languages/german.lang

```diff
@@ -661,7 +662,7 @@ msgid "Disc 2"
 msgstr "Disk 2"
 
 msgid "Disc Artwork"
-msgstr ""
+msgstr "Disk Cover"
```

Comment:
`Cover` ist not German.

Reply:
There is also no proper translation for "Artwork" in this case. "Cover" would be a translation that fits. "Cover" is also Germanized and can also be found in the Duden ([here](https://www.duden.de/rechtschreibung/Cover)).